TourMate/TourMate/Frontend/Trip/Views/TripView/TripView.swift

```diff
@@ -44,32 +44,62 @@ struct TripView: View {
-                                    .clipped()
-                            } placeholder: {
-                                Color.gray
+                TabView {
+                    ScrollView {
+                        VStack {
+                            Text(dateString)
+                                .font(.headline)
+                                .frame(maxWidth: .infinity, alignment: .leading)
+                                .padding([.bottom, .horizontal])
+                            AttendeesView(viewModel: viewModel)
+
+                            if let imageUrl = viewModel.trip.imageUrl {
+                                AsyncImage(url: URL(string: imageUrl)) { image in
+                                    image
+                                        .resizable()
+                                        .scaledToFill()
+                                        .frame(height: 200, alignment: .center)
+                                        .clipped()
+                                } placeholder: {
+                                    Color.gray
+                                }
                             }
+
+                            PlansListView(tripId: viewModel.trip.id, tripViewModel: viewModel)
                         }
-                        Text("Attendees")
-                            .font(.headline)
-                            .frame(maxWidth: .infinity, alignment: .leading)
-                            .padding([.bottom, .horizontal])
+                    }
+                    .tabItem {
+                        Image(systemName: "list.dash")
+                        Text("List View")
+                    }
 
-                        AttendeesView(viewModel: viewModel)
+                    ScrollView {
+                        VStack {
+                            Text(dateString)
+                                .font(.headline)
+                                .frame(maxWidth: .infinity, alignment: .leading)
+                                .padding([.bottom, .horizontal])
+                            AttendeesView(viewModel: viewModel)
 
-                        PlansListView(tripId: viewModel.trip.id, tripViewModel: viewModel)
+                            if let imageUrl = viewModel.trip.imageUrl {
+                                AsyncImage(url: URL(string: imageUrl)) { image in
+                                    image
+                                        .resizable()
+                                        .scaledToFill()
+                                        .frame(height: 200, alignment: .center)
+                                        .clipped()
+                                } placeholder: {
+                                    Color.gray
+                                }
+                            }
+
+                            PlansCalendarView(tripId: viewModel.trip.id, tripViewModel: viewModel)
+                        }
+
+                    }
```

Comment:
I think we should abstract out the scroll views? It looks the same as the one above.